rosa/cli.py

```diff
@@ -56,6 +56,7 @@ def parse_help(rosa_cmd="rosa"):
     output_flag_str = "-o, --output"
     auto_answer_yes_str = "-y, --yes"
     auto_mode_str = "-m, --mode"
+    billing_model_str = "--billing-model"
```

Comment:
`billing-model` have options, not like --yes for example, since this is a client we need to support all other option as well.